xbmc/cores/VideoPlayer/VideoPlayerRadioRDS.cpp

```diff
@@ -1230,11 +1146,7 @@ unsigned int CDVDRadioRDSData::DecodeRTPlus(uint8_t *msgElement, unsigned int le
   if (m_RTPlus_iToggle == 0)    // RTplus tags V2.1, only if RT
     return 10;
 
-  if (!m_RTPlus_Present)
-  {
-    m_currentInfoTag->SetPlayingRadiotextPlus(true);
-    m_RTPlus_Present = true;
-  }
+  m_currentInfoTag->SetPlayingRadiotextPlus(true);
```

Comment:
Capital `T` for `Text`?

I know this is from the existing code, but not reason not to fix it while your there.

Reply:
Changed.